src/types/view/index.ts

```diff
@@ -55,7 +56,7 @@ export interface ViewSubmitAction {
   view: ViewOutput;
   api_app_id: string;
   token: string;
-  trigger_id: string; // undocumented
+  trigger_id?: string; // undocumented
```

Comment:
This change, as well as the similar change in BlockActions type is to reflect the fact that now in certain cases (aka when the payload is sent with `FunctionContext`, the trigger_id will NOT be included).